configure.ac

```diff
@@ -84,6 +84,7 @@ AS_IF([test "x$enable_turn_off_mihawkcpld_access" != "xyes"],
       AC_DEFINE_UNQUOTED([MIHAWKCPLD_DEVICE_ACCESS], ["$MIHAWKCPLD_DEVICE_ACCESS"], [Turn off MIHAWKCPLD hardware access])
 )
 
+
```

Comment:
any reason for these new empty lines?  (if no other changes are required in this file, i don't care if they are removed)

Reply:
@spinler I'll fix it.